src/routes/HomePage/HomePage.js

```diff
@@ -1,11 +1,68 @@
 import React from 'react';
 import { Section } from '../../components/Utils/Utils';
 import './HomePage.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCat, faStar, faPaw } from "@fortawesome/free-solid-svg-icons";
 
 export default function HomePage() {
+
+  const renderImage = () => {
+    return (
+      <div className='homeImageContainer'>
+        <img src={process.env.PUBLIC_URL + '/homeimage.JPG'}
+          alt="HomeImage" width="100%" object-fit="cover" object-position="50% 50%"></img>
+        <div id="message" className="font-weight-bold text-white">
+          FIND YOUR PURRFECT PET<br />
+        </div>
+      </div>
+    );
+  };
+
+  const renderOverview = () => {
+    return (
+      <div className="container text-center m-3">
+        <div className="row d-flex p-2">
+
+          <div className="col-sm border rounded m-1 p-2">
+            <FontAwesomeIcon icon={faPaw} className="fa-4x text-info" />
+            <h3>Our Vision</h3>
+            <p>Intently sniff hand sleep on my human's head. Lick face hiss at owner, pee a lot, and meow repeatedly scratch at fence purrrrrr eat muffins and poutine until owner comes back chase little red dot someday it will be mine!</p>
```

Comment:
These are great haha, did you come up with these?

Did we want to update these to actual sentences? Or figure it out at a later date?

Reply:
I was thinking we can figure it out later! I just googled cat ipsum haha

Reply:
Agree, I'm imaging we'll have a general "polish up" story. We can add that then